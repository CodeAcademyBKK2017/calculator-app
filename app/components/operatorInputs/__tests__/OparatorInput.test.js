import 'react-native';
import OPINPUT from '../OperatorInputs.component';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <OPINPUT />
  );
  expect(tree).toBeDefined();
});

it('BaseView: renders correctly', () => {
  const tree = renderer.create(<OPINPUT />).toJSON();
  expect(tree).toMatchSnapshot();
});