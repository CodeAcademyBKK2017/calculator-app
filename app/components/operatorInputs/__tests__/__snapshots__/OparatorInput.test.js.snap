// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseView: renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  keyboardType="numeric"
  onChangeText={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "borderWidth": 1,
      "fontSize": 50,
      "width": "30%",
    }
  }
  value=""
/>
`;
